package.json

```diff
@@ -11,13 +11,15 @@
     "enzyme-adapter-react-16": "^1.6.0",
     "history": "^4.7.2",
     "jwt-decode": "^2.2.0",
+    "moment": "^2.22.2",
```

Comment:
@MuhanguziDavid Do you need this when you have `react-moment`?

Reply:
@samuelbwambale , I installed `react-moment` that package was probably installed as a dependency

Reply:
Cool